packages/core/app/mirage/routes/dashboard.js

```diff
@@ -59,25 +50,22 @@ export default function() {
       dashboards = dashboards.all();
     }
 
-    dashboards.models.forEach(dashboard => assignWidgets(dashboard, dashboardWidgets));
-    dashboards.models.forEach(dashboard => assignDeliveryRules(dashboard, deliveryRules));
     return dashboards;
   });
 
-  this.post('/dashboards', function({ dashboards, db }) {
+  this.post('/dashboards', function({ dashboards, users }) {
     let attrs = this.normalizedRequestAttrs(),
-      dashboard = dashboards.create(attrs);
+      dashboard = dashboards.create(attrs),
+      author = users.find(dashboard.authorId);
 
     // Update user with new dashboard
-    db.users.update(dashboard.authorId, {
-      dashboards: dashboard.author.dashboards.concat([Number(dashboard.id)])
-    });
+    author.update(dashboards, author.dashboards.add(dashboard));
 
     // Init properties
-    db.dashboards.update(dashboard.id, {
+    dashboard.update({
       widgetIds: [],
-      createdOn: moment.utc().format(TIMESTAMP_FORMAT),
-      updatedOn: moment.utc().format(TIMESTAMP_FORMAT)
+      createdOn: null,
```

Comment:
what is the reason for this change?

Reply:
![screen shot 2019-02-04 at 3 19 06 pm](https://user-images.githubusercontent.com/23023478/52237884-4afce480-2890-11e9-9c34-78cf81d4246b.png)

Some tests were comparing to the returned dashboard model and expecting null values. But now that I look into the reports route, it has the same test but returns the timestamp. I'm not sure why the test is getting null values for `createdOn` and `updatedOn` for reports but not dashboards, so I'll look into that further.

Reply:
That was probably done b/c we do a full object compare and would not know the expected dates in the test.